library/kotlin/src/io/envoyproxy/envoymobile/Envoy.kt

```diff
@@ -53,4 +54,22 @@ class Envoy @JvmOverloads constructor(
   fun isTerminated(): Boolean {
     return runner.state == Thread.State.TERMINATED
   }
+
+  override fun startStream(request: Request, responseHandler: ResponseHandler): StreamEmitter {
```

Comment:
We don't have `startStream` anymore, just the two `send` functions: https://github.com/lyft/envoy-mobile/blob/master/library/swift/src/Client.swift